src/utilities/pyswice_ck_utilities.py

```diff
@@ -22,7 +22,7 @@
 from Basilisk.utilities import RigidBodyKinematics, macros
 
 
-def ckWrite(handle, time, MRPArray, avArray, startSeg, sc = -62, rf = "J2000"):
+def ckWrite(handle, time, MRPArray, avArray, startSeg, sc=-62, rf="J2000"):
```

Comment:
while we are here, let's make these variable names more verbose. Something like "spacecraft_id" or "reference_frame" .
We should also snake case everywhere